src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -8384,23 +8387,59 @@ instruct binOps_HF_reg(fRegF dst, fRegF src1, fRegF src2)
   ins_pipe(fp_dop_reg_reg_s);
 %}
 
-instruct min_max_HF_reg(fRegF dst, fRegF src1, fRegF src2)
+instruct min_HF_reg(fRegF dst, fRegF src1, fRegF src2, rFlagsReg cr)
```

Comment:
Ah, the max/min_HF part of the work seems duplicate https://github.com/openjdk/jdk/pull/24047?

Reply:
Ah, I'll remove the part. Thanks!